app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
+    if user&.authenticated?(:activation, params[:id]) && !user.activated?
```

Comment:
user&. syntax này là sao em

Reply:
user&.autheticated? tương tự với user && user.authenticated? . Đây là toán tử điều hướng an toàn(safe navigation) trong Ruby. Cũng là cách viết mà Ruby Style Guide khuyên dùng.